Source/WebCore/css/parser/CSSParserContext.h

```diff
@@ -98,6 +98,7 @@ struct CSSParserContext {
     bool masonryEnabled { false };
     bool cssNestingEnabled { false };
     bool cssPaintingAPIEnabled { false };
+    bool cssTextUnderlinePositionLeftRightEnabled { false };
```

Comment:
Do we need this?

Reply:
Was getting compile errors without it

Reply:
ah I see overflowClipEnabled is in there, looks fine.